draft-ietf-taps-interface.md

```diff
@@ -1039,8 +1039,7 @@ Note that if an application Requires the use of temporary addresses, the
 resulting Connection cannot use IPv4, as temporary addresses do not exist in
 IPv4.
 
-
-### Parallel Use of Multiple Paths {#parallel-multipath}
+### Parallel Use of Multiple Paths {#multipath-mode}
```

Comment:
How about "Support for Multiple Parallel Paths"?

Reply:
Or "management of multiple parallel paths"...?